Dockerfile

```diff
@@ -0,0 +1,87 @@
+#
+# Dockerfile for an image with the currently checked out version of zipline installed. To build:
+#
+#    docker build -t quantopian/zipline .
+# 
+# To run the container:
+#
+#    docker run -v=/path/to/your/notebooks:/projects -p 8888:8888/tcp --name zipline -it quantopian/zipline
```

Comment:
I think the error I was seeing is actually more interesting than I thought.  If I run the container with

`docker run -p 8888:8888 --name zipline -it quantopian/zipline`, when I try to open a notebook I get this in the browser:

![screenshot from 2016-03-29 12-13-57](https://cloud.githubusercontent.com/assets/1588357/14115159/d8fe8064-f5a7-11e5-9de2-1c3a6eab27a0.png)

In the javascript console, I have a bunch of errors that look like:

```
Failed to load resource: net::ERR_INSECURE_RESPONSE
main.min.js?v=cee9d5d…:13001Object
main.min.js?v=cee9d5d…:13003API request failed (404): error
https://localhost:8888/api/terminals?_=1459267991926 Failed to load resource: net::ERR_INSECURE_RESPONSE
main.min.js?v=cee9d5d…:13001Object
main.min.js?v=cee9d5d…:13003API request failed (404): error
https://localhost:8888/api/sessions?_=1459267991927 Failed to load resource: net::ERR_INSECURE_RESPONSE
main.min.js?v=cee9d5d…:13001Object
main.min.js?v=cee9d5d…:13003API request failed (404): error
https://localhost:8888/api/terminals?_=1459267991928 Failed to load resource: net::ERR_INSECURE_RESPONSE
main.min.js?v=cee9d5d…:13001Object
main.min.js?v=cee9d5d…:13003API request failed (404): error
```